components/x11/xf86-video-cirrus/Makefile

```diff
@@ -11,18 +11,19 @@
 
 #
 # Copyright 2015 Alexander Pyhalov
+# Copyright 2017 Michal Nowak
 #
 
 include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME= xf86-video-cirrus
-COMPONENT_VERSION= 1.5.2
+COMPONENT_VERSION= 1.5.3
 COMPONENT_REVISION=1
```

Comment:
Stupid page just shifted when I clicked, so just fix the component revision and it should be fine.